Recipe google analytics (#22648)
gatsbyjs/gatsby: docs/docs/recipes/deploying-your-site.md

### Additional resources

- [Hosting on Cloudflare](/docs/deploying-to-cloudflare-workers)

> ## Setting up google analytics

Reviewer: ```suggestion
## Setting up Google Analytics
```

> ### Using gatsby-plugin-google-analytics to track the activity of your Gatsby Site

Reviewer: ```suggestion
### Using gatsby-plugin-google-analytics
```

> gatsby-plugin-google-analytics is a very popular plugin that provides insights in the access patters of your website

Reviewer: ```suggestion
`gatsby-plugin-google-analytics` is a popular plugin that provides insights into how users access your website. You can use it to track site activity.
```

### Prerequisites

> - A [Gatsby site](/docs/quick-start) with `gatsby-config.js` and an `index.js` page

Reviewer: ```suggestion
- A [Gatsby site](/docs/quick-start) with a `gatsby-config.js` file and an `index.js` page
```

- The [Gatsby CLI](/docs/gatsby-cli) installed

> - A Domain ex: [AWS](https://aws.amazon.com/getting-started/tutorials/get-a-domain/)

Reviewer: ```suggestion
- A Domain from your provider of choice,  e.g. [AWS](https://aws.amazon.com/getting-started/tutorials/get-a-domain/)
```

> #### Verify the domain in search.google.com

Reviewer: ```suggestion
### Verify the domain in search.google.com
```

> 1. [Verify the domain](https://search.google.com/search-console/not-verified) by clicking on **Search Property** > **Add Property**. Type in your domain, and press Continue.

Reviewer: ```suggestion
1. Navigate to the [Google search console]((https://search.google.com/search-console/not-verified)) to verify the domain by clicking on **Search Property** > **Add Property**. Type in your domain and press Continue.
```

> 2. Create a **TXT** record in the DNS configuration for example [on AWS](https://www.entrustdatacard.com/knowledgebase/how-to-create-a-txt-record-on-amazon-aws-route-53-for-entrust-email-validation-method)

Reviewer: ```suggestion
2. Add a **TXT** record to your DNS configuration. Follow the directions for your provider, or refer to [Google docs](https://support.google.com/a/answer/183895?hl=en).
```

> #### Linking the domain to Google analytics admin

Reviewer: ```suggestion
### Linking the domain to Google Analytics admin
```

> 1. Log in to [Analytics](https://analytics.google.com/analytics/)

Reviewer: ```suggestion
1. Log into [Google Analytics](https://analytics.google.com/analytics/).
```

2. Click Admin.

> 3. Select **Create Property** in Property column

Reviewer: ```suggestion
3. Select **Create Property** in the Property column.
```

> 4. Choose **Web**

Reviewer: ```suggestion
4. Choose **Web**.
```

5. Fill in the details and click Create.

> #### Getting the analytics tracking id

Reviewer: ```suggestion
### Getting your Google Analytics `Tracking ID`
```

> 1. Sign in to your Analytics account.

Reviewer: ```suggestion
1. Sign in to your Google Analytics account.
```

2. Click Admin.
3. Select an account from the menu in the ACCOUNT column.
4. Select a property from the menu in the PROPERTY column.
5. Under PROPERTY, click Tracking Info > Tracking Code. Your Tracking ID is displayed at the top of the page.

> #### Using the id in the plugin

Reviewer: ```suggestion
### Using the ID in the plugin
```

> 1. `npm install --save gatsby-plugin-google-analytics`

Reviewer: ```suggestion
1. Run `npm install gatsby-plugin-google-analytics` in your terminal.
```

> 2. in gatsby-config.js add

Reviewer: ```suggestion
2. Add the following to your `gatsby-config.js` file.
```

> ```js

Reviewer: ```suggestion
```javascript:title="gatsby-config.js"
```
